README.md

```diff
@@ -74,6 +74,7 @@ Here is a list of sbt commands for daily development:
 > findbugs                                 # Produce findbugs report in target/findbugs
 > jacoco:cover                             # Report the code coverage of tests to target/jacoco folder
 > jcheckStyle                              # Run check style
+> scalafmt                                 # Reformat Scala codes
```

Comment:
https://github.com/lucidsoftware/neo-sbt-scalafmt/blob/1.14/README.md#usage

> scalafmt       # format compile sources
> test:scalafmt  # format test sources
> sbt:scalafmt   # format .sbt source

The `scalafmt` command format only compile sources

Reply:
https://github.com/msgpack/msgpack-java/pull/443/files#r153711477